crates/pera-core/src/dwallet_mpc/mpc_instance.rs

```diff
@@ -144,10 +171,8 @@ impl DWalletMPCInstance {
     ) -> PeraResult<()> {
         let party_id = authority_name_to_party_id(message.authority, &epoch_store)?;
         if self.pending_messages.contains_key(&party_id) {
```

Comment:
To better self-document the code, consider this:
```suggestion
        let already_received_message_from_party = self.pending_messages.contains_key(&party_id);
        if already_received_message_from_party {
```

Reply:
Not crucial imo